server/protocols/webadmin/webadmin-mailbox/src/main/java/org/apache/james/webadmin/routes/UserQuotaRoutes.java

```diff
@@ -90,6 +98,16 @@ private static Optional<Integer> intQueryParameter(Request request) {
                     "strictly positive optional integer", USERS_PER_SECOND), e);
             }
         }
+
+        private static List<QuotaComponent> getQuotaComponent(Request request) {
+            return Optional.ofNullable(request.queryParamsValues(QUOTA_COMPONENT))
+                .map(quotaComponentStrings -> Arrays.stream(quotaComponentStrings)
+                    .map(quotaComponentString -> Optional.ofNullable(QUOTA_COMPONENT_MAP.get(quotaComponentString))
```

Comment:
IMO we should ignore case when using Map for store component key,
`.toLowerCase` or `.toUpperCase` is fine
If not, we should update the document. 
The example in document `&quotaComponent=MAILBOX&quotaComponent=JMAP_UPLOADS` will not work

Reply:
I saw your update in the new commits, but still consider ignoring case if want

Reply:
Cases should be ignored with query params, I agree

Reply:
fixed